t/script/banes/sendtopassthrough.t

```diff
@@ -51,7 +51,7 @@ FixMyStreet::override_config {
     $c = CGI::Simple->new($req->content);
 
     is $c->{service_request_id}[0] eq '248', 1, "Passthrough service_request_id is set as the report's passthrough id";
-
+    is $c->{service_code}[0] eq 'passthrough-confirm_graffiti@example.org', 1, "service_code given email address to send to passthrough";
```

Comment:
Again, please can we have `is A, B` rather than `is A eq B, 1` because that produces nicer output. Wherever this happens